suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml

```diff
@@ -178,7 +178,7 @@ tests:
       destroy-cluster: false
       abort-on-fail: true
   - test:
-      name: Upgrade along with I/O's
+      name: Upgrade along with IOs
```

Comment:
@manasagowri Sorry i don't know that using test name it's going to create log file
```

> Traceback (most recent call last):
> 
>   File "run.py", line 1026, in <module>
> 
>     rc = run(args)
> 
>   File "run.py", line 741, in run
> 
>     tc["log-link"] = configure_logger(unique_test_name, run_dir)
> 
>   File "/home/jenkins-build/workspace/rhceph-cron-pipeline-test-executor/utility/utils.py", line 458, in configure_logger
> 
>     _handler = logging.FileHandler(test_logfile)
> 
>   File "/usr/lib64/python3.6/logging/__init__.py", line 1032, in __init__
> 
>     StreamHandler.__init__(self, self._open())
> 
>   File "/usr/lib64/python3.6/logging/__init__.py", line 1061, in _open
> 
>     return open(self.baseFilename, self.mode, encoding=self.encoding)
> 
> FileNotFoundError: [Errno 2] No such file or directory: "/ceph/cephci-jenkins/cephci-run-DUJTEE/Upgrade_along_with_I/O's_0.log

 ```

yes due to this PR https://github.com/red-hat-storage/cephci/commit/6b6f850acab00aed5346cbf1280b8f17425327b1
Tests getting failed for that / in IO's, But I have updated like this name for other upgrade suites 
Please refer to the attached PR here and make all other suites like this name
Thanks for the quick fix.

Reply:
Thanks Sunil. I made the changes in all relevant test suites.